packages/react-scripts/scripts/eject.js

```diff
@@ -220,8 +220,21 @@ inquirer
     }
 
     if (fs.existsSync(paths.yarnLockFile)) {
```

Comment:
Shouldn't the whole `if statement` be commented out and it just runs `spawnSync('npm', ['install'], { stdio: 'inherit' });` every time?